caddyhttp/fastcgi/setup.go

```diff
@@ -27,6 +27,10 @@ import (
 	"github.com/mholt/caddy/caddyhttp/httpserver"
 )
 
+//default timeout for fastcgi
+//specific timeouts can be disabled by setting it to 0 in config
+var defaultTimeout = 60 * time.Second
```

Comment:
I think these comments can be deleted -- the first line is obvious, and the second one isn't necessarily true if something changes elsewhere in the code; instead that should be noted in the docs on the website which is a different matter.

Reply:
Ok, that was my question) How can i contribute changes to docs?